server/jetstream_test.go

```diff
@@ -23925,3 +23925,32 @@ func TestJetStreamConsumerStartSequenceNotInStream(t *testing.T) {
 		require_Equal(t, consumer.asflr, 10)
 	}()
 }
+
+func TestInterestStreamWithDuplicateMessages(t *testing.T) {
```

Comment:
Is the test name ok? I think it has to be `TestJetStreamInterestStreamWithDuplicateMessages` for it to run

Reply:
https://github.com/nats-io/nats-server/blob/main/scripts/runTestsOnTravis.sh#L41

Reply:
Corrected this and another case.